### PR TITLE
Add instructions for developing web UI

### PR DESCRIPTION
**Pull request by costanzo**

### CODE REVIEW

File: js/apps/admin-ui/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <artifactId>keycloak-admin-ui</artifactId>
 
     <name>Keycloak Admin UI</name>
-    <description>The user inferface to administrate the Keycloak server.</description>
+    <description>The user interface to administrate the Keycloak server.</description>
```
Comment (jonkoops) [inline js/apps/admin-ui/pom.xml:17]: Could you also fix this typo in the account console as well?

File: js/apps/keycloak-server/README.md
Comment (jonkoops) [inline js/apps/keycloak-server/README.md:1]: Let's split this work off into a separate PR, it is unrelated to the other changes here.

**Review by jonkoops — DISMISSED** [review]

**Review by ssilvert — DISMISSED** [review]

File: js/apps/keycloak-server/README.md
Comment (costanzo) [reply js/apps/keycloak-server/README.md:1]: I have put the typo fix in another PR: https://github.com/keycloak/keycloak/pull/32636.

File: js/apps/admin-ui/pom.xml (hunk 1 above)
Comment (costanzo) [reply js/apps/admin-ui/pom.xml]: Sure

**Review by edewit — DISMISSED** [review]

File: js/apps/keycloak-server/README.md [hunk 2]
```diff
@@ -41,6 +41,16 @@ If you want to run with a local Quarkus distribution of Keycloak for development
 pnpm start --local
 ```
 
+If you want to run with `IDELauncher`, make sure you set the following environment variables.
```
Comment (jonkoops) [inline js/apps/keycloak-server/README.md:44]: This is not specific to `IDELauncher`, but pertains to all instances where a Keycloak server is started.
```suggestion
If you want to run Keycloak standalone (without the script) against the Vite development server, you can set the following environment variables to achieve the same result:
```

File: js/apps/keycloak-server/README.md [hunk 3]
```diff
@@ -41,6 +41,16 @@ If you want to run with a local Quarkus distribution of Keycloak for development
...
+
+```sh
+export KC_ACCOUNT_VITE_URL=http://localhost:5173
+export KC_ADMIN_VITE_URL=http://localhost:5174
+export KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci
+```
+
+This environment variables will make your Keycloak admin/account UI retrieve web resource from the local Vite server.
```
Comment (jonkoops) [inline js/apps/keycloak-server/README.md:52]: There should be no need to explain this again.

```suggestion
```

File: js/apps/keycloak-server/README.md [hunk 4]
```diff
@@ -41,6 +41,16 @@ If you want to run with a local Quarkus distribution of Keycloak for development
 pnpm start --local
 ```
 
+If you want to run with `IDELauncher`, make sure you set the following environment variables.
+
+```sh
+export KC_ACCOUNT_VITE_URL=http://localhost:5173
+export KC_ADMIN_VITE_URL=http://localhost:5174
+export KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci
```
Comment (jonkoops) [inline js/apps/keycloak-server/README.md:49]: Let's not add the `export` statement here, most people will configure these in their IDE and this makes for easier copy-pasting.
```suggestion
KC_ACCOUNT_VITE_URL=http://localhost:5173
KC_ADMIN_VITE_URL=http://localhost:5174
KC_FEATURES=login2,account3,admin-fine-grained-authz,transient-users,oid4vc-vci
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/keycloak-server/README.md (hunk 2 above)
Comment (costanzo) [reply js/apps/keycloak-server/README.md]: make sense to me

**Comment by ssilvert** [comment]
@jonkoops Please take another look at this one.

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
Thanks for the contribution @costanzo!
